pretix_sepadebit/signals.py

```diff
@@ -59,6 +68,70 @@ def register_csv(sender, **kwargs):
     return DebitList
 
 
+@receiver(register_mail_placeholders, dispatch_uid="payment_sepadebit_placeholders")
+def register_mail_renderers(sender, **kwargs):
+
+    ph = [SimpleFunctionalMailTextPlaceholder(
+          'due_date', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.due.date, sample=date.today()
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'account', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('account', " "), sample="Max Mustermann"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'iban', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('iban')[0:4] + " **** " + sepadebit_payment.info_data.get('iban')[-4:], sample="DE02 **** 2051"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'bic', ['sepadebit_payment'], lambda sepadebit_payment: sepadebit_payment.info_data.get('bic'), sample="BYLADEM1001"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'reference',
+            ['sepadebit_payment'],
+            lambda sepadebit_payment: sepadebit_payment.info_data.get('reference'),
+            lambda event: f'{event.settings.reference_prefix + "-" if event.settings.reference_prefix else ""}{event.slug.upper()}-XXXXXXX'
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'creditor_id', ['sepadebit_payment', "event"], lambda sepadebit_payment, event: event.settings.creditor_id, sample="DE98ZZZ09999999999"
+        ),
+        SimpleFunctionalMailTextPlaceholder(
+            'creditor_name', ['sepadebit_payment', "event"], lambda sepadebit_payment, event: event.settings.creditor_name, sample="DE98ZZZ09999999999"
+        )]
+
+    return ph
+
+
+
+@receiver(signal=periodic_task, dispatch_uid="payment_sepadebit_send_payment_reminders")
+def send_payment_reminders(sender, **kwargs):
+    with scopes_disabled():
+        dd = SepaDueDate.objects.filter(reminded=False).filter(remind_after__lt=datetime.now(timezone.utc)).filter(payment__order__event__plugins__contains='pretix_sepadebit').select_related('payment').select_related('payment__order').prefetch_related('payment__order__event')
```

Comment:
1. The line is way to long to read
2. you don't need to call filter so often.
3. Now that we're not looping over *events* but (basically) *payments* I rescind my previous comment and think we don't need to check for the plugin to be active. Disabling a payment provider plugin *after* you created payments already is unsupported and causes enough more urgent bugs already. 
4. There's a nice ``now()`` function in ``django.utils.timezone`` :)
5. We should however check that the payment is still active, we don't want to send notifications for canceled payments

Proposal:

```suggestion
        dd = SepaDueDate.objects.filter(
            reminded=False,
            remind_after__lt=now(),
            payment__state=OrderPayment.PAYMENT_STATE_CONFIRMED
        ).select_related('payment', 'payment__order').prefetch_related('payment__order__event')
```